source/release_notes/index.rst

```diff
@@ -7,6 +7,7 @@ Percona Server for MongoDB |version| Release Notes
 .. toctree::
    :maxdepth: 1
 
+   5.0.9-8
```

Comment:
it seems suspicious that new line is not similar to the previous ones. Is this what was intended?

Reply:
Yes, all this fuss is to have release dates included in the index. With 5.0.9-8, I've added the release date to the heading so it automatically appears in the index.